osm_fieldwork/update_xlsform.py

```diff
@@ -133,10 +163,10 @@ def create_survey_group(name: str) -> dict[str, pd.DataFrame]:
         {
             "type": ["begin group"],
             "name": [name],
-            "label::English(en)": [name],
-            "label::Swahili(sw)": [name],
-            "label::French(fr)": [name],
-            "label::Spanish(es)": [name],
+            "label::english(en)": [name],
```

Comment:
Very nice addition!

Just checking that forms definitely work with the lowercase label? Did you manage to test anywhere? 🙏

(I don't see why not, as it should match the (en) key, but need to be sure!)

Reply:
Yes it does, i tested it